### DISCUSSION TITLE
[Projects beta] Error while removing an issue

### DISCUSSION THREAD

**Discussion by sertxudev** [community/community#10993] [Projects and Issues]
If you want to delete an issue from a project, you simply select the issue and press the "del" key.
But instead of deleting it, the message "Title cannot be deleted" appears.

![image](https://user-images.githubusercontent.com/22801379/152499516-a3f7fa83-86cf-4cf0-b51f-1301eddc98c2.png)

So I have to use the dropdown and click the "Delete from project" button.

![image](https://user-images.githubusercontent.com/22801379/152499653-cb56bfd9-3769-4112-9ea9-4a02169a2ee7.png)

**Comment by MakePixelsWork** [comment]
I have the same problem, even with Drafts.

**Comment by sertxudev** [comment]
Selecting the title shows the message:
![image](https://user-images.githubusercontent.com/22801379/210543360-7f33378e-3715-4b9b-922a-eefc6aff8c94.png)

If you select the row by clicking on the row number works as expected:
![image](https://user-images.githubusercontent.com/22801379/210543479-e8d69361-4d98-4022-81e9-e2af4c582019.png)
